test/integration/puppetlabs/general_puppet/general_puppet_int_test.clj

```diff
@@ -259,7 +259,7 @@
 (deftest ^:integration static-file-content-endpoint-test-no-code-content-command
   (logging/with-test-logging
    (testing "the /static_file_content endpoint errors if :code-content-command is not set"
-     (bootstrap/with-puppetserver-running
+     (bootstrap/with-puppetserver-running-with-mock-jrubies
```

Comment:
Yeah, basically the process of reviewing this PR for me is going to be looking at each one of the calls to this macro one by one and reasoning about why it is/isn't sane.  So maybe a docstring-ish approach isn't totally bananapants?